helpers/custom.js

```diff
@@ -0,0 +1,116 @@
+import StyleDictionary from "style-dictionary";
```

Comment:
This is great! I'm wondering if we can further split this up like we do in hw-admin with an index.js file, and separate files for custom filters, transforms and filter groups. Thoughts?

Reply:
i considered this also but then decided against it because they all sort of work together, and are called within `transformGroups` - so i thought it might be nice to have them not separated? 

but like i said i also had this thought initially so if thats how we wanna go i'm cool with it